test/storage.js

```diff
@@ -0,0 +1,21 @@
+/* global module */
```

Comment:
Wondering if this file should be in a different directory, so it's not confused as a potential test? Something like `test/fixtures` maybe?